edgelet/iotedge/test-files/config/ca-certs-est/edged.toml

```diff
@@ -7,6 +7,7 @@ edge_ca_key = "aziot-edged-ca"
 trust_bundle_cert = "aziot-edged-trust-bundle"
 auto_reprovisioning_mode = "OnErrorOnly"
 homedir = "/var/lib/aziot/edged"
+allow_privileged_docker_containers = true
```

Comment:
Since privileged has a specific meaning wrt containers, and we are using the flag to mean more than that, can we use a more generic name? 
Say something like allow_elevated_permissions or something like that?

Reply:
makes sense, I'll change it to `allow_elevated_docker_permissions `